modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,248 @@
+
+* This update supports the matrix feature to organize tasks in a pipeline. For example, you can define a single task in your pipeline which executes multiple times based on the matrix parameters. This update is part of an alpha preview feature.
+
+* With this update, you can specify the `stdoutConfig` and `stderrConfig` parameters when defining pipeline steps. Defining these parameters helps to capture standard output and standard error associated with steps to local files.
+
+* With this update, you can add variables in the `steps[].onError` event handler, for example, `$(params.CONTINUE)`.
+
+* With this update, you can use the output from the `finally` task in the `PipelineResults` definition. For example, `$(finally.<pipelinetask-name>.result.<result-name>)`, where `<pipelinetask-name>` denotes the pipeline task name and `<result-name>` denotes the result name.
+
+* This update supports task-level resource requirements for a task run.
+
+* With this update, you do not need to recreate parameters that are shared based on their name between a pipeline and the defined tasks. This update is part of an alpha preview feature.
+
+* This update adds support for remote resolution, such as built-in git, cluster, bundle, and hub resolvers.
+
+
+[id="triggers-new-features-1-9_{context}"]
+=== Triggers
+
+* This update adds the `Interceptor` custom resource definition to define `NamespacedInterceptor`. You can use `NamespacedInterceptor` in the kind section of interceptors reference in triggers or in the `EventListener` specification.
+
+* This update enables `CloudEvents`, which do not explicitly require the stable API feature during trigger processing.
+
+* With this update, you can configure the webhook port number when defining a trigger.
+
+* This update supports trigger `eventID` to be used as input to `TriggerBinding`.
+
+* This update supports validation and rotation of certificates for the `ClusterInterceptor` server. 
+** Triggers perform certificate validation for core interceptors and rotate a new certificate to `ClusterInterceptor` when its certificate expires. 
+
+
+[id="cli-new-features-1-9_{context}"]
+=== CLI
+
+* This update adds support for a pipeline run with minimal embedded status. 
+
+* This update supports showing annotations in the `describe` command.
+
+* This update supports showing pipeline, tasks, and finally timeout in the `pr describe` command.
+
+* This update adds flags to provide pipeline, tasks, and finally timeout in the `pipeline start` command.
+
+* This update adds the status of workspaces in the `describe` command of a task and pipeline.
+
+* This update adds the `timestamps` flag to show logs with a timestamp.
+
+* This update adds a new flag `--ignore-running-pipelinerun`, which ignores the deletion of `TaskRun` associated with `PipelineRun`.
+
+* With this update, you can use the parameters `qps` and `bursts` to configure the Tekton controller performance.
```

Comment:
this is hardcoded, not configurable, and it is for k8s api server not controller